### ISSUE TITLE
An unexpected error occurred on Navigator start-up

### ISSUE THREAD

**Issue by pjgoodall** [ContinuumIO/anaconda-issues#2668]
Navigator Error

An unexpected error occurred on Navigator start-up

Report

Please report this issue in the anaconda issue tracker

Main Error

byte indices must be integers or slices, not str
Traceback

Traceback (most recent call last):
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/exceptions.py", line 75, in exception_handler
    return_value = func(*args, **kwargs)
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/app/start.py", line 115, in start_app
    window = run_app(splash)
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/app/start.py", line 58, in run_app
    window = MainWindow(splash=splash)
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/widgets/main_window.py", line 160, in __init__
    self.api = AnacondaAPI()
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/api/anaconda_api.py", line 1205, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/api/anaconda_api.py", line 65, in __init__
    self._conda_api = CondaAPI()
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/api/conda_api.py", line 1622, in CondaAPI
    CONDA_API = _CondaAPI()
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/api/conda_api.py", line 340, in __init__
    self.set_conda_prefix()
  File "/home/peter/anaconda3/lib/python3.6/site-packages/anaconda_navigator/api/conda_api.py", line 489, in set_conda_prefix
    self.ROOT_PREFIX = info['root_prefix']
TypeError: byte indices must be integers or slices, not str

**Comment by goanpeca** [comment]
Closing as duplicate of #1837

Please update to the latest version of Navigator.

On Navigator click on the update button on the top right of the interface or
on the terminal type

```
$ conda update anaconda-navigator
```

**Comment by pjgoodall** [comment]
All packages were up-to-date. Anaconda was installed fresh yesterday.
Just tried suggestion - same error.

Cheers...

**Comment by LBsh** [comment]
I have tried uninstall and reinstall the whole package but nothing works. Running as administrator does not help.

**Comment by rodrigoporcionato** [comment]
hi, check if exist a site package in Python. Normaly in this path : C:\Users\youruser\AppData\Roaming\Python and delete the folder. This it work with me.

**Comment by pjgoodall** [comment]
I'm on Ubuntu 16.04 LTS

Sorry I forgot to mention my platform...

**Comment by klapi85** [comment]
I got the same problem with Ubuntu 14.04 and Anaconda 4.4.0 with Python 3.6 installer (64-Bit).

**Comment by klapi85** [comment]
I've noticed there is a problem while using the "fish" terminal. Bash goes ok.

**Comment by goanpeca** [comment]
We only have official support for bash, although it may or may not work on other shells.

**Comment by taymourniazi** [comment]
An unexpected error occurred on Navigator start-up
psutil.AccessDenied
So I used several commands like
-	Conda update anaconda-navigator
-	Conda update navigator-updater
-	Conda update --prefix C:\Users\asd\Anaconda2 anaconda
It all doesn’t fix the issue only the following work fine
-	Anaconda-navigator --reset

**Comment by mohammadraasel** [comment]
on teminal type:
conda update -n base conda

**Comment by carmamir** [comment]
@mohammadraasel 

Already guys confirmed that this is not a solution !

Doesn't work at all. Fresh install, Windows.

**Comment by csoja** [comment]
@carmamir  You tried _anaconda-navigator --reset_  from an Anaconda Prompt window?

**Comment by ranand985** [comment]
@csoja  Thanks a lot. The command you suggested worked on my system (Mac).

**Comment by anushreddy** [comment]
@csoja Worked like a charm. Thanks

**Comment by freyesBci** [comment]
@csoja After searching everywhere this solved everything! Thanks

**Comment by DennisScuba** [comment]
I tried all mentioned possibilities and couldn't fix it, has someone another idea?

**Comment by ESSIENEKPO** [comment]
I type Anaconda-navigator --reset on Anaconda command prompt on windows 8.1 and it reset Anaconda Navigator. After reset, it prompted for update to version 1.8.4 which I did. It prompted again for another update to version 1.8.7 and I followed the screen, after that Anaconda Navigator loaded and works fine. Note, it will prompt you to close Anaconda navigator as the update progresses. Thanks for this forum.
